setup GUI update

SamuelMagda opened the pull request:
- updated GUI
- created a BAT file for installing python 310 and a virtual environment where all the requirements are installed
- created a BAT file that runs the GUI pictureminator - that is fully functional
- updated readme
- updated requirements

SamuelMagda committed:
setup GUI update

- updated GUI
- created a BAT file for installing python 310 and a virtual environment where all the requirements are installed
- created a BAT file that runs the GUI pictureminator - that is fully functional

Tehnik762 reviewed (changes requested):
I don't like the idea - that we will stick to
1. EXE file in Git 
2. Version of OS someone is using. 

The right way to do it
1. Find out - what we are having - Win, Linux or Mac
2. Download right version and install it. 
3. Change the bat files to the appropriate running scripts.

SamuelMagda committed:
pictureminator_manager.bat

- only one bat for intalling python and venv
- a folder with separated scripts
- no python.exe, and its not installing localy, rather globaly with path

SamuelMagda committed:
Update Readme.md
